zerver/lib/markdown/tabbed_sections.py

```diff
@@ -137,10 +138,16 @@ def generate_content_blocks(self, tab_section: Dict[str, Any], lines: List[str])
     def generate_nav_bar(self, tab_section: Dict[str, Any]) -> str:
         li_elements = []
         for tab in tab_section["tabs"]:
-            li = NAV_LIST_ITEM_TEMPLATE.format(
-                data_language=tab.get("tab_name"), name=TAB_DISPLAY_NAMES.get(tab.get("tab_name"))
-            )
-            li_elements.append(li)
+            tab_name = tab.get("tab_name")
+            try:
+                li = NAV_LIST_ITEM_TEMPLATE.format(
+                    data_language=tab_name, name=TAB_DISPLAY_NAMES[tab_name]
+                )
+            except KeyError as e:
+                raise ValueError(f"Display name for the tab '{tab_name}' is missing") from e
```

Comment:
Can we tweak this error message to mention how to add it?  Perhaps:

"Tab {tab_name} not present in TAB_DISPLAY_NAMES in zerver/lib/markdown/tabbed_sections.py"

Reply:
Got it